x/examples/website-wrapper-app/wrapper_app_project/.scripts/build.mjs

```diff
@@ -224,14 +224,11 @@ function resolveTemplateArguments(
     result.domainList = [result.entryDomain];
   }
 
-  if (typeof smartDialerConfig === "string") {
-    result.smartDialerConfig = smartDialerConfig.replaceAll('"', '\\"');
-  } else if (typeof smartDialerConfig === "object") {
-    result.smartDialerConfig = JSON.stringify(smartDialerConfig).replaceAll(
-      '"',
-      '\\"',
-    );
-  }
+  result.smartDialerConfig = Buffer.from(
+    typeof smartDialerConfig === "object"
```

Comment:
Ambiguous types hurts readability. Please enforce the config to be a string.
That means calling stringfy on the default config as needed.